docs/guide/setup-examples/remote_dev.md

```diff
@@ -0,0 +1,189 @@
+This steps should help working remotely with Sublime Text.
+They are meant to be incremental,
+just setting up SFTP will go a long way.
+
+# My workflow
+
+I have all my code on my laptop (a Mac),
+edit locally and automatically push the files to my server (a Linux machine).
+I never edit "tracked" files on the server directly.
+I sometimes modify untracked files
+or config files on the server using `rmate` (see below).
+
+# SSH config
+
+Create a `~/.ssh/config` file,
+which allows reusing existing connection.
+This mean that you will have to authenticate only once.
+
+```
+Host server
+  Hostname SERVER_ADRESS
+  User MY_USERNAME
+  IdentityFile YOUR_KEY eg: ~/.ssh/id_rsa
+  AddKeysToAgent yes
+  UseKeychain yes
+  ControlPath ~/.ssh/ssh-%r@%h:%p
+  ControlMaster auto
+  ServerAliveInterval 240
+  Port 22
```

Comment:
All these options are optional and unless they are required, users shouldn't be asked to copy random congifuration options into their setup that they don't need or understand. That's my ideal, at least

Reply:
In my case I need the ControlPath, ControlMaster because I have 2-factor authentication.
I split the instructions in two, and added more details